src/_posts/2020-09-17-cassandra-usage-report-2020.markdown

```diff
@@ -0,0 +1,94 @@
+
+
+    "I saw zero downtime at global scale with Apache Cassandra. That's a powerful statement to make. For our business that’s quite crucial." - Practitioner, London
+
+**Cassandra adoption is correlated with organizations in a more advanced stage of digital transformation.**
+
+People from organizations that self-identified as being in a “highly advanced” stage of digital transformation were more likely to be using Cassandra (26%) compared with those in an “advanced” stage (10%) or “in process” (5%). 
+
+**Optionality, security, and scalability are among the key reasons Cassandra is selected by practitioners.**
+
+The top reasons practitioners use Cassandra for mission critical apps are “good hybrid solutions” (62%), “very secure” (60%), “highly scalable” (57%), “fast” (57%), and “easy to build apps with” (55%). 
+
+**A lack of skilled staff and the challenge of migration deters adoption of Cassandra.**
+
+Thirty-six percent of practitioners currently using Cassandra for mission critical apps say that a lack of Cassandra-skilled team members may deter adoption. When asked what it would take for practitioners to use Cassandra for more applications and features in production, they said “easier to migrate” and “easier to integrate.”
+
+### Methodology
+
+**Sample.** The survey consisted of 1,404 interviews of IT professionals and executives, including 901 practitioners which is the focus of this usage report, from April 13-23, 2020. Respondents came from 13 geographies (China, India, Japan, South Korea, Germany, United Kingdom, France, the Netherlands, Ireland, Brazil, Mexico, Argentina, and the U.S.) and the survey was offered in seven languages corresponding to those geographies. While margin of sampling error cannot technically be calculated for online panel populations where the relationship between sample and universe is unknown, the margin of sampling error for equivalent representative samples would be +/- 2.6% for the total sample, +/- 3.3% for the practitioner sample, and +/- 4.4% for the executive sample. 
+
+To ensure the highest quality respondents, surveys include enhanced screening beyond title and activities of company size (no companies under 100 employees), cloud IT knowledge, and years of IT experience.
+
+**Rounding and multi-response.** Figures may not add to 100 due to rounding or multi-response questions.
+
+### Demographics
+
+Practitioner respondents represent a variety of roles as follows: Dev/DevOps (52%), Ops/Architect (29%), Data Scientists and Engineers (11%), and Database Administrators (8%) in the Americas (43%), Europe (32%), and Asia Pacific (12%). 
+
+![Cassandra roles](src/img/blog-post-usage-report-2020/image1.jpg "image_tooltip")
```

Comment:
@loganloganlogan , all the images need to be `/img/blog-post-usage-report-2020/…` (i.e. the url part begin with `/img/`)

(I had to correct that late in the last blog post here https://github.com/loganloganlogan/cassandra-website/commit/ed73115251f2f4ffd83cb3edfb0008e6b0764f8a#diff-bedb8fd638d536c9b7ebfd8dabd55b4b )

Reply:
+1, I could not see them as part of the preview as well, so let's just double-check the rendering before publishing it.

Reply:
Noted for next time.

Reply:
I'm still struggling w/ rendering locally using make serve; might have a fix that will allow me to view moving forward.

Reply:
They were still missing the initial slash, i.e. `/img/…` vs `img/…`
I've fixed this in the merge.